test.c

```diff
@@ -3377,6 +3378,125 @@ test_parse_url (void)
   }
 }
 
+struct find_crlf_test {
+  const char* name;
+  const char *p;
+  int offset;
+  int len;
+  int should_find;
+  int position;
+};
+
+const struct find_crlf_test find_crlf_tests[] =
+{ {.name="long string without crlf"
+  ,.p="                                                "  /* 16*3 */
+  ,.offset=0
+  ,.len=48
+  ,.should_find=0
+  ,.position=0
+  }
+  ,{.name="long string with crlf"
+  ,.p="                                               \n"  /* 16*3 */
+  ,.offset=0
+  ,.len=48
+  ,.should_find=1
+  ,.position=47
+  }
+  ,{.name="short string without crlf"
+  ,.p="  "  /* 16*3 */
```

Comment:
What is the significance of this same code comment for each newly added test?

Reply:
nothing significant, just a badly done copy and paste :P